services/genesis/transcription.service.ts

```diff
@@ -46,7 +51,7 @@ class MedicalTranscriptionAPI implements MedicalTranscription {
     transcription: string,
     template?: string,
     language?: string,
-  ): Promise<string> {
+  ): Promise<any> {
```

Comment:
This could be a record(Record<string, number | string | null>) if we are expecting a object